### DISCUSSION TITLE
Anisotropic voxel analysis for FIB Tomo

### DISCUSSION THREAD

**Discussion by JoeQuinnPNNL** [NREL/MATBOX_Microstructure_analysis_toolbox#28] [Ideas]
Hello,

I am interested in using MATBOX for FIB Tomography 3D reconstructions. The issue I have is that this analysis would require anisotropic voxels which is not currently supported in MATBOX. (Slice thickness is dependent on the FIB milling thickness and not on the SEM field of view). Enabling MATBOX analysis for 3D reconstructions would have an wide variety of applications. For example we could measure battery properties (porosity, tortuosity, or surface area) for real electrode architectures using FIB-SEM. This would be useful for the multiple studies in the SCP consortium. 

Joe Quinn
PNNL

**Comment by fussegli** [comment]
Hi Joe,
The tool has been initially developed for X-ray CT 3D volumes, for which voxels are isotropic. However, with more users going to MATBOX, with various imaging methods such as FIB-SEM, the voxel isotropy assumption can be quite limiting. I will add the option to rescale the volume from anisotropic to isotropic voxel size for the next MATBOX version currently in preparation.
